utils/gitlog2changelog.py

```diff
@@ -15,7 +16,20 @@
     rev_range = "%s..HEAD" % base
 
 # Execute git log with the desired command line options.
-fin = os.popen("git log --summary --stat --no-merges --date=short %s" % rev_range, "r")
```

Comment:
The fin variable is used later on. You should see the failure in the Create or check a release draft workflow which is failing on this PR (once you fix the process versus fin issue - see below).

Reply:
this was changed by @kpolchow in a previous change